conf/local.conf.nonostree.append

```diff
@@ -0,0 +1,28 @@
+
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+
+PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"
+
+# Store systemd logs in persistent storage
+#
+# It greatly helps diagnosing issues on testing devices but should be
+# carefully weighted against file system usage and flash device wear for
+# production systems. Please refer to systemd's docs for more details
+IMAGE_INSTALL_append += " systemd-journald-persistent"
+
+# Set resource limits for aktualizr service
+#
+# Uncomment these lines to change the default parameters.
+#
+#RESOURCE_CPU_WEIGHT_pn-aktualizr = "100"
+#RESOURCE_MEMORY_HIGH_pn-aktualizr = "100M"
+#RESOURCE_MEMORY_MAX_pn-aktualizr = "80%"
+
+IMAGE_INSTALL_append += " aktualizr-resource-control"
```

Comment:
yes, of course, we can reuse local.conf.systemd.append but I think that it can add more confusion.
The  local.conf.systemd.append is intended to be used in the context of sota.bbclass, and actually, "conf.append" does not configure system-d.
I think it's not a good idea just to reuse "conf.append", because the different configurations files may be intended to different bb classes, does it make sense?

Reply:
But the point here is that these `.append` files should be transparent to the user. The `envsetup` script does the work of automatically appending them to `local.conf`, so I'm not sure who would get confused by the end result.

> The local.conf.systemd.append is intended to be used in the context of sota.bbclass

I disagree; it's just some logging and resource limiting that isn't specific to `sota.bbclass`.

> actually, "conf.append" does not configure system-d

True, but it's configuration that should only be used with systemd, which we are assuming applies in this case as well. If someone doesn't want it, they can remove it from their `local.conf` along with everything else systemd-specific.